.gitlab/fuzzing.yml

```diff
@@ -0,0 +1,38 @@
+variables:
+  # Use the same base image as benchmarks as we know it should be able to build the fuzzers
+  BASE_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:libddwaf
+
+run-fuzzing:
+  stage: fuzzing
+  tags: ["runner:apm-k8s-tweaked-metal"]
+  image: $BASE_CI_IMAGE
+  timeout: 30m
+  # rules:
+  #   - if: '$CI_COMMIT_REF_NAME == "master" && $CI_PIPELINE_SOURCE == "schedule"'
+  #   - if: '$CI_COMMIT_REF_NAME == "master" && $CI_PIPELINE_SOURCE == "push"'
+  #   - if: '$CI_PIPELINE_SOURCE == "web"'
+  #     when: manual
+  before_script: |
```

Comment:
Feel free to open a PR on the benchmarking platform repo to update this image and avoid reinstalling every time.